### PR TITLE
Updated documentation with schema examples

### PR DESCRIPTION
**Pull request by MHove89**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,6 +178,80 @@ Some meta-data is required for this to work. As you can see in the example, we h
...
+			"type": "partial",
+			"input": "{root}",
+			"alias": "migrationPageMetaData"
+		}
+	}
+}
+
+```
+
+##### Partial schema example with meta-data
```
Comment (emilras) [inline README.md:223]: @MHove89 This one is the "migrationPageMetaData", right? If so, perhaps that should described unambiguously?

**Review by emilras — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (MHove89) [reply README.md:223]: I agree. Will do :)

Comment (MHove89) [reply README.md]: @emilras I have added the change. Do you see more that we should do before doing the PR?

Comment (emilras) [reply README.md]: @MHove89 Nope. It's good :-)

**Pull request merged**